scripts/deploy.sh

```diff
@@ -130,6 +132,12 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
   # setup the hosts and build the basic containers
   run_ansible setup-hosts.yml
 
+  # Apply security hardening if apply_security_hardening is true
+  # (By default, apply_security_hardening is false and this will be skipped)
```

Comment:
I think this should use the existing pattern for controlling execution by defining a variable that is used by this script, existing examples include DEPLOY_MAAS and DEPLOY_CEPH.